core/types/rlp_payload.libevm.go

```diff
@@ -46,13 +46,21 @@ func RegisterExtras[
 		HeaderHooks
 		*H
 	},
+	BodyExtra any, BodyExtraPtr interface {
```

Comment:
same here, can this be changed to `B any, BPtr ...` if body and block get merged?